app/AppKernel.php

```diff
@@ -44,6 +46,7 @@ public function registerBundles()
             new OpenOrchestra\LogBundle\OpenOrchestraLogBundle(),
             new OpenOrchestra\UserAdminBundle\OpenOrchestraUserAdminBundle(),
             new OpenOrchestra\MediaAdminBundle\OpenOrchestraMediaAdminBundle(),
+            new OpenOrchestra\WorkflowAdminBundle\OpenOrchestraWorkflowAdminBundle(),
```

Comment:
WorflowAdminBundle and FunctionBundle are not consistent. Can you choose one of them ?